app/renderer/components/navigation/navigationBar.js

```diff
@@ -115,6 +125,42 @@ class NavigationBar extends React.Component {
       this.props.bookmarked
   }
 
+  // BEM Level: navigator__navigationButtonContainer
```

Comment:
By adding a comment like that, it would be easy to understand what the code structure is like while you separate components into functions.